src/server/integrations/handleGitHubWebhooks.js

```diff
@@ -0,0 +1,48 @@
+import crypto from 'crypto';
+import schema from 'server/graphql/rootSchema';
+import {graphql} from 'graphql';
+
+// TODO when this is all legit, we'll map through the queries & use the ASTs instead of the strings
+const eventLookup = {
+  organization: {
+    member_added: {
+      getVars: ({membership: {user: {login: userName}}, organization: {login: orgName}}) => ({userName, orgName}),
+      query: `
+        mutation GitHubAddMember($userName: ID! $orgName: ID!) {
+          githubAddMember(userName: $userName, orgName: $orgName)
+        }
+      `
+    },
+    member_removed: {
+      getVars: ({membership: {user: {login: userName}}, organization: {login: orgName}}) => ({userName, orgName}),
+      query: `
+        mutation GitHubRemoveMember($userName: ID! $orgName: ID!) {
+          githubRemoveMember(userName: $userName, orgName: $orgName)
+        }
+      `
+    }
+  }
+};
+
+export default async (req, res) => {
+  res.sendStatus(200);
+  const event = req.get('X-GitHub-Event');
+  const hexDigest = req.get('X-Hub-Signature');
+  const [shaType, hash] = hexDigest.split('=');
+  const {body} = req;
+  const myHash = crypto
+    .createHmac(shaType, process.env.GITHUB_CLIENT_SECRET)
```

Comment:
any vulns you can imagine by sending GITHUB our client secret? the only reason i use this is because i don't wanna keep track of 1 more env variable, especially when it changes for each env (local, dev, staging, prod).